### PR TITLE
Code Quality: Change active pane border thickness in high contrast

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

- Increased border thickness for active pane in high contrast mode

image

### CODE REVIEW

File: src/Files.App/Views/Shells/ModernShellPage.xaml [hunk 1]
```diff
@@ -84,11 +100,13 @@
 				<VisualState x:Name="ShellBackgroundFocusOnState">
 					<VisualState.Setters>
 						<Setter Target="RootGrid.Background" Value="{ThemeResource App.Theme.FileArea.BackgroundBrush}" />
+						<Setter Target="RootGrid.BorderThickness" Value="{ThemeResource Local.ActivePane.BorderThickness}" />
```
Comment (yaira2) [inline src/Files.App/Views/Shells/ModernShellPage.xaml:103]: We need a way to only apply this resource when dual pane is enabled.

Comment (yaira2) [reply src/Files.App/Views/Shells/ModernShellPage.xaml:103]: It seems that we'll need another Visual State for when Dual Pane is turned off.

Comment (0x5bfa) [reply src/Files.App/Views/Shells/ModernShellPage.xaml:103]: Yeah this is for background so we can create ShellBorderThicknessFocusOnState and OffState.

**Review by 0x5bfa — COMMENTED** [review]
LGTM, code wise.

**Pull request merged**